build.gradle

```diff
@@ -120,7 +120,7 @@ jacocoTestReport {
     }
 }
 
-String version = '1.4.2'
+String version = '1.4.3'
```

Comment:
This should be minor bump since we are adding new library here. Kind of feature. So Increment to 1.5.0